bbb-common-message/src/main/scala/org/bigbluebutton/common2/util/UrlValidationUtil.scala

```diff
@@ -0,0 +1,16 @@
+package org.bigbluebutton.common2.util
+
+import java.net.URL
+
+object UrlValidationUtil {
+  def isValidUrl(url: String): Boolean = {
+    if (url == null) return false
+    try {
+      val parsedUrl = new URL(url)
+      val protocol = parsedUrl.getProtocol
+      protocol == "http" || protocol == "https"
+    } catch {
+      case _: MalformedURLException => false
```

Comment:
[error] /tmp/build/bbb-fsesl-akka_3.0.0~beta.4+0-git.0_jammy/bbb-common-message/src/main/scala/org/bigbluebutton/common2/util/UrlValidationUtil.scala:13:15: not found: type MalformedURLException